book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rental
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = rental
+  end
```

Comment:
- [ ] Kindly create a method `add_rental` in the `Book` class that takes in `person`, `book`, and `date` as parameters. The method should create a new `Rental`. Remember that `Person` and `Book` have a _**many to many**_ relationship through `Rental`. This should be mirrored in the `Book` class. You can read this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) to get a better understanding. 👍

Reply:
I think what I understood from the example is that `person` and `book` has `many to many` relationships and when a a new `rental` object is created it at the same time we will add the `rental` to `@rental` of the `person` and `book`. That's we added` person.rental << self` and `book.rental << self` as part of the constructor

Reply:
Hi @awaisanwar544 , you did that correctly but remember that `Book` and `Person` should have a relationship with `Rental`. This means that the `Book` and `Person` classes should be able to keep track of their rental and that can be implemented by creating a new `Rental`. All you have to do is create a method that creates a new `Rental` object. 👍